base/src/main/java/com/austindroids/austinfeedsme/di/modules/EventFilterPresenterModule.kt

```diff
@@ -0,0 +1,14 @@
+package com.austindroids.austinfeedsme.di.modules
+
+import com.austindroids.austinfeedsme.eventsfilter.EventFilterActivity
+import com.austindroids.austinfeedsme.eventsfilter.EventFilterContract
+import dagger.Binds
+import dagger.Module
+
+@Module
+abstract class EventFilterPresenterModule {
```

Comment:
What an odd name for something that provides a `View`.

Reply:
Do you have a good example of how to provide view interfaces that are implemented by an activity?

Reply:
Hmm assuming [Vasiliy Zukanov is wrong and it's ok for Activities/Fragments to be views instead of controllers](https://www.youtube.com/watch?v=oOARoQr4H5U), I'd probably rather call it `__MvpViewModule`, considering you are providing the view, and not the presenter here.